src/style.css

```
@@ -1,3 +1,112 @@
 body {
   margin: 0;
+  font-family: "Roboto", "Helvetica", sans-serif;
+  --color-blue: rgba(0, 172, 220, 1);
+  --color-black: rgba(6, 11, 53, 1);
+  --color-grey: rgba(97, 96, 112, 1);
+
+}
+
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  margin: auto;
+  padding: 33px 17px 17px;
+  box-sizing: border-box;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 35px auto 0;
+  color: var(--color-black);
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-grey);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: auto;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.stars__star {
+  background: url(../src/images/star.svg) no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.card__number {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.stars--1 > .stars__star:nth-child(-n + 1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  background: url(../src/images/star-active.svg) no-repeat;
+  background-position: center;
+}
+
+.card__size {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price {
+  color: var(--color-grey);
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__text {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--color-black);
+}
```

1. remove margin
2. rename
```suggestion
.card__price-text {
  color: var(--color-grey);
  font-weight: 400;
  font-size: 12px;
  line-height: 18px;
}

.card__price-value {
  font-size: 16px;
  line-height: 18px;
  font-weight: 700;
  color: var(--color-black);
}
```